README.md

```diff
@@ -117,15 +117,15 @@ This repository contains many companion libraries to write integration tests wit
+| Individual test execution timing                                                          | ✔️       | ✔️                                                           | [Lib](https://hackage.haskell.org/package/tasty-stats)           |
 | Test suite execution timing                                                               | ✔️       | ✔️                                                           | ✔️                                                                |
-| Helpful output to find slow tests                                                         | ✔️       | [Lib](http://hackage.haskell.org/package/hspec-slow)        | C                                                                |
+| Helpful output to find slow tests                                                         | ✔️       | ✔️                                                           | C                                                                |
 | Coloured output                                                                           | ✔️       | ✔️                                                           | ✔️                                                                |
 | Colourless output                                                                         | ✔️       | ✔️                                                           | ✔️                                                                |
 | `ppShow` instead of `show` for counterexample output                                      | ✔️       | ✖️                                                           | [Lib](https://hackage.haskell.org/package/tasty-hedgehog)        |
 | `show` for counterexample output                                                          | 🚧      | ✔️                                                           | [Lib](https://hackage.haskell.org/package/tasty-hunit)           |
-| Fancy Unicode output                                                                      | ✔️       | ✖️                                                           | C                                                                |
-| Unicode-free output                                                                       | 🚧      | ✖️                                                           | ✔️                                                                |
+| Fancy Unicode output                                                                      | ✔️       | ✔️                                                           | C                                                                |
+| Unicode-free output                                                                       | 🚧      | ✔️                                                           | ✔️                                                                |
```

Comment:
I think this was always the case; please prove me wrong.

Reply:
I don't remember "fancy unicode output" in hspec, but I trust you if you tell me it's there.